### ISSUE TITLE
Add an example of MovementScript

### ISSUE THREAD

**Issue by DolphinGamerYT** [kinsi55/CS_BeatSaber_Camera2#17]
Hey,
I'm trying to create a movement script, and I've been looking into it, but it's just not working. I'm also just using replays and fpfc, but I don't think it should be the problem. Let me attach you my movement script, and the camera using it. 

zoomincam.json
`{
  "visibleObjects": {
    "Walls": "Visible",
    "Debris": true,
    "UI": false,
    "Avatar": true,
    "Floor": true,
    "CutParticles": true,
    "Notes": "Visible"
  },
  "type": "Positionable",
  "worldCamVisibility": "HiddenWhilePlaying",
  "previewScreenSize": 2.0,
  "FOV": 70.0,
  "layer": 5,
  "antiAliasing": 2,
  "renderScale": 1.3,
  "viewRect": {
    "x": 0.0,
    "y": 0.0,
    "width": -1.0,
    "height": -1.0
  },
  "FPSLimiter": {
    "fpsLimit": 240
  },
  "ModmapExtensions": {
    "moveWithMap": true,
    "autoOpaqueWalls": true,
    "autoHideHUD": false
  },
  "Follow360": {
    "enabled": false,
    "smoothing": 10.0
  },
  "targetPos": {
    "x": 0.0,
    "y": 5.0,
    "z": 0.0
  },
  "targetRot": {
    "x": 0.0,
    "y": 0.0,
    "z": 0.0
  },
  "MovementScript": {
    "scriptList": [
      "zoomin"
    ],
    "fromOrigin": true,
    "enableInMenu": true
  }
}`

zoomin.json (Movement Script)
`{
    "syncToSong": false,
    "loop": true,
    "Frames": [
        {
            "position": {
                "x": 0,
                "y": 5,
                "z": -1
            },
            "rotation": {
                "x": 0.0,
                "y": 0.0,
                "z": 0.0
            },
            "duration": 5,
            "holdTime": 1
        },
        {
            "position": {
                "x": 0,
                "y": 0.2,
                "z": -1
            },
            "rotation": {
                "x": 0.0,
                "y": 0.0,
                "z": 0.0
            },
            "duration": 5,
            "holdTime": 1
        }
    ]
}
`

Some logs too:
[_latest_camera.log](https://github.com/kinsi55/CS_BeatSaber_Camera2/files/6253612/_latest.log)
[_latest.log](https://github.com/kinsi55/CS_BeatSaber_Camera2/files/6253595/_latest.log)

Also, when a scene is configured to show a camera with a movement script, does the movement script start over, or does the movement start when the song starts playing, and when switching the scene, the movement is in the middle?

**Comment by DolphinGamerYT** [comment]
Btw, I think I've set the wrong label. Sorry if I did 😓

**Comment by DolphinGamerYT** [comment]
Tried to look at sourcecode, and I think I saw that frames should be under "Movements", not "Frames". By doing that, the issue changes:

[_latest.log](https://github.com/kinsi55/CS_BeatSaber_Camera2/files/6253624/_latest.log)

**Comment by DolphinGamerYT** [comment]
Wait, nvm ^^

Solved it by adding a third keyframe, but the error was printing `Movement scripts must contain at least two keyframes`. Pretty weird.

**Comment by kinsi55** [comment]
Oops, seems like I messed up the loading of scripts

```
[ERROR @ 00:14:57 | Camera2] Failed to load Movement script zoomin.json
[ERROR @ 00:14:57 | Camera2] System.Exception: Movement scripts must contain at least two keyframes
[ERROR @ 00:14:57 | Camera2]   at Camera2.Managers.MovementScriptManager.LoadMovementScripts (System.Boolean reload) [0x00064] in <385dfe21506141029338edceda9bc843>:0 
```

Obviously, you do have two keyframes, but with how the code works right now you actually need three, so if you add a third keyframe it should work.

>Also, when a scene is configured to show a camera with a movement script, does the movement script start over, or does the movement start when the song starts playing, and when switching the scene, the movement is in the middle?

If you use `enableInMenu`, as you do, and use the same camera ingame, the movement will continue where it left off. If you want it to reset and start from 0 you can enable `syncToSong` in the movement script

>Tried to look at sourcecode, and I think I saw that frames should be under "Movements", not "Frames". By doing that, the issue changes:

frames is actually the correct key, movements is what camera+ uses.

**Comment by DolphinGamerYT** [comment]
Thank you so much!

**Comment by kinsi55** [comment]
Fixed in v0.3.26 https://github.com/kinsi55/CS_BeatSaber_Camera2/releases/latest
